app/ui-react/syndesis/src/modules/integrations/components/editor/choice/ChoiceStepPage.tsx

```diff
@@ -1,15 +1,16 @@
 import {
   createConditionalFlow,
   FlowKind,
-  getFlow,
+  getFlow, getPreviousIntegrationStepWithDataShape,
```

Comment:
`getPreviousIntegrationStepWithDataShape` should go on the next line.  Ideally you'd want to install whatever extension adds `prettier` support to your IDE, then it should pick up the rules and automatically apply them.  I'd say run `yarn format` and commit the result but testing locally I see that there's a few other unrelated files that haven't been formatted using `prettier` at the moment.

Reply:
Thanks, I have now installed some IntelliJ plugin - lets see how that goes